Premiere/15.0/index.d.ts

```diff
@@ -2010,26 +2010,40 @@ declare class ComponentParamCollection {
 
 declare class ComponentParam {
   readonly displayName: string
-  addKey(): boolean
+  addKey(time: Time, updateUI?: boolean): boolean
   areKeyframesSupported(): boolean
-  findNearestKey(): object
-  findNextKey(): object
-  findPreviousKey(): object
+  /**
+   * threshold in ticks
+   */
+  findNearestKey(timeToCheck: Time, threshold: number): object
```

Comment:
No need for the comment, and `ticks` usually come as string if I'm not mistaken
```suggestion
  findNearestKey(timeToCheck: Time, thresholdInTicks: string): object
```